.github/workflows/cd.yaml

```
@@ -4,9 +4,33 @@ on:
   push:
     branches:
       - master
+      - eks-test-2
 
 jobs:
+  pre-clean:
+    # Delete existing S3 folder and K8s namespace
+    # if they already exist from a previous run.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+      - name: Set current date as env variable $TODAY
+        run: echo "TODAY=$(date +'%y-%m-%d')" >> $GITHUB_ENV
+      - env:
+          CLUSTER: zq-test
+          NAMESPACE: ci-${{ github.run_number }}
+          ZQD_DATA_URI: s3://zq-ci-test/${{ env.TODAY }}.${{ github.run_number }}
+        run: |
+          set -x
```

For simplicity, if you need to use environment variables, like the `TODAY` and the `env` section, you could do so inside the run block:

```suggestion
        run: |
          set -x
          ZQD_DATA_URI=s3://zq-ci-test/$(date +'%y-%m-%d').${{ github.run_number }}
          ...
```

---

done